test/libsolidity/semanticTests/strings/empty_storage_string.sol

```diff
@@ -0,0 +1,69 @@
+contract C {
+
+    string emptyString;
+    string emptyString2 = "";
+    string nonEmptyString = "This is a non empty string";
+    string str = "Another string";
+    error EmptyError(string);
```

Comment:
The previous version also was emitting an event, wasn't it? I'd add that too for better coverage.